pkg/util/ipvs/ipvs_test.go

```diff
@@ -380,3 +383,37 @@ func TestFrontendDestinationString(t *testing.T) {
 		}
 	}
 }
+
+func TestGetRequiredIPVSMods(t *testing.T) {
+	Tests := []struct {
+		name          string
+		kernelVersion *version.Version
+		want          []string
+	}{
+		{
+			name:          "KernelPriorTo4.19",
+			kernelVersion: version.MustParseGeneric("4.18"),
+			want:          []string{ModIPVS, ModIPVSRR, ModIPVSWRR, ModIPVSSH, ModNfConntrackIPV4},
+		},
+		{
+			name:          "Kernel4.19",
+			kernelVersion: version.MustParseGeneric("4.19"),
+			want:          []string{ModIPVS, ModIPVSRR, ModIPVSWRR, ModIPVSSH, ModNfConntrack},
+		},
+	}
+	for _, test := range Tests {
+		t.Run(test.name, func(t *testing.T) {
+			got := GetRequiredIPVSMods(test.kernelVersion)
+			if len(got) != len(test.want) {
+				t.Errorf("GetRequiredIPVSMods() = %v for kenel version: %s, want %v", got, test.kernelVersion, test.want)
+			}
+			sort.Strings(got)
+			sort.Strings(test.want)
+			for i := range test.want {
```

Comment:
Use reflect.DeepEqual here since the slice is already sorted?

Reply:
Fixed.